langchain/document_loaders/web_base.py

```diff
@@ -117,7 +117,7 @@ async def _fetch(
                         headers=self.session.headers,
                         ssl=None if self.session.verify else False,
                     ) as response:
-                        return await response.text()
+                        return await response.content.read()
```

Comment:
@rhabal seems this returns bytes not string, what was the error you were seeing related to this? and can we save it for a different pr?

Reply:
+1, agreed we should take this up in a new PR. 

This loader is used in a lot of places.

More context on this change (and motivation re: unicode error) is needed.